book/chapters/git-advanced-branches.qmd

```diff
@@ -85,50 +72,37 @@ However, if you now try to switch branches to main (click on "cakes" and choose
 ### Merging the changes into the main branch
 
 How can we now "merge" the new recipe into the main branch?
-In Git merging two branches in the web browser can be done by so-called "Pull requests" or PRs. Pull requests are a way to change, review, and merge code in a Git repository.
+The recommended and most convenient way is to work with so-called "Pull requests" or PRs. Pull requests are a feature of online git platforms (GitHub, Codeberg, etc.). Pull requests offer the possibility to review, discuss and update proposed changes before merging them into the main branch.
 
 To create a Pull request, click on the tab "Pull Requests" and then on the button "New pull request".
 
-We can choose now the "base" branch, the branch we would like to update and the "compare" branch, the branch with which we would like to update the "base".
+We can choose now the "base" branch, the target branch that should be updated with changes that have been applied to the "compare" branch.
 
-Select as "base" your main branch and **adjust the repository**. Since our repository is a "Fork", GitHub assumes we would like to merge our changes into the repository we forked our repository from. We can adjust that by clicking on "base repository" and o our own repository. Next we set the base branch to "main". Now we need to choose the  compare branch and set that to "cakes".
+Select as "base" your main branch and **adjust the repository**. Since our repository is a "Fork" of an existing, GitHub assumes we would like to merge our changes into the repository that we forked our repository from. We can adjust that by clicking on "base repository" and then on our own repository. Next, we set the base branch to "main". Now we need to choose the compare branch and set that to "cakes".
```

Comment:
```suggestion
Select as "base" your main branch and **adjust the repository**. Since our repository is a "Fork" of an existing repository, GitHub assumes we would like to merge our changes into the repository that we forked our repository from. We can adjust that by clicking on "base repository" and then on our own repository. Next, we set the base branch to "main". Now we need to choose the compare branch and set that to "cakes".
```